tests/unit_tests/test_routes.py

```diff
@@ -48,8 +48,6 @@ def test_get_id(self):
         response = self.app.post(
             "/kml", data=self.kml_string, content_type="application/vnd.google-earth.kml+xml"
         )
-        self.assertEqual(response.status_code, 201)
-        self.assertEqual(response.content_type, "application/json")
```

Comment:
Why removing those two tests ? If the post failed, then we should not continue testing...